sqlglot/expressions.py

```diff
@@ -3961,6 +3961,11 @@ class Between(Predicate):
 class Bracket(Condition):
     arg_types = {"this": True, "expressions": True}
 
+    @property
+    def output_name(self):
+        if self.expressions:
```

Comment:
Why was this check necessary? The `expressions` arg is mandatory for `Bracket`.

Reply:
Fair enough, removed.